code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -93,6 +93,8 @@
 	name = "\improper NF10 'Saber' SMG"
 	desc = "A rejected prototype three-round burst 9mm submachine gun, designated 'SABR'. Surplus of this model are bouncing around armories of Nanotrasen Space Stations. Has a threaded barrel for suppressors."
 	icon_state = "saber"
+	lefthand_file = 'icons/mob/inhands/guns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/guns_righthand.dmi'
```

Comment:
These are the default `/obj/item/gun` lefthand and righthand files. To my knowledge neither `/obj/item/gun/projectile` nor `/obj/item/gun/projectile/automatic` change this
```suggestion
```